src/Images.jsx

```diff
@@ -115,7 +116,7 @@ class Images extends React.Component {
             </div>
         );
         const columns = [
-            { title: image.RepoTags ? image.RepoTags[0] : "", header: true },
+            { title: image.RepoTags ? image.RepoTags[0] : "<none>:<none>", header: true },
```

Comment:
Is "<none>" some kind of standard convention? It's not i18n'ed, and also looks a bit weird. I wondered how the CLI shows them, but I don't see any:
```
❱❱❱ podman images --all
REPOSITORY                                     TAG     IMAGE ID      CREATED      SIZE
docker.io/library/busybox                      latest  c7c37e472d31  2 weeks ago  1.45 MB
registry.fedoraproject.org/f32/fedora-toolbox  32      7e4d1b69a358  3 weeks ago  394 MB
docker.io/library/debian                       sid     e1d741d6634c  5 weeks ago  123 MB
```

Could this perhaps be `_("Unnamed intermediate image")` or `_("Intermediate image")` or so, with italic formatting?

Reply:
Yes, that is how CLI shows that, thats why I went with that:
```
REPOSITORY                         TAG     IMAGE ID      CREATED            SIZE
localhost/foobar                   latest  66fe3398edde  About an hour ago  17.5 MB
<none>                             <none>  afdffa38735b  About an hour ago  11.7 MB
docker.io/library/fedora           latest  a368cbcfa678  9 days ago         189 MB
docker.io/library/ubuntu           latest  adafef2e596e  13 days ago        76.3 MB
localhost/tnuro                    5       27c9a4e9a337  2 weeks ago        1.05 GB
localhost/tnuro                    latest  27c9a4e9a337  2 weeks ago        1.05 GB
<none>                             <none>  5f359173d668  2 weeks ago        1.05 GB
<none>                             <none>  beafbcb1a253  2 weeks ago        1.05 GB
<none>                             <none>  a30842c5884e  2 weeks ago        1.05 GB
<none>                             <none>  2784fd2faf41  2 weeks ago        1.05 GB
<none>                             <none>  8b2bf089d928  2 weeks ago        1.05 GB
<none>                             <none>  e29e1eb8a53b  2 weeks ago        1.05 GB
<none>                             <none>  d0a4984abc21  2 weeks ago        1.05 GB
<none>                             <none>  c1b358886b1a  2 weeks ago        1e+03 MB
<none>                             <none>  744ca788c9e0  2 weeks ago        991 MB
<none>                             <none>  b1a5b6945399  2 weeks ago        991 MB
<none>                             <none>  3dc83a0616a6  2 weeks ago        593 MB
<none>                             <none>  46a0609ac99f  2 weeks ago        199 MB
docker.io/library/mongo            latest  2b2cc1f48aed  4 weeks ago        392 MB
docker.io/library/alpine           latest  a24bb4013296  7 weeks ago        5.85 MB
registry.fedoraproject.org/fedora  31      a7a37f74ff86  2 months ago       199 MB
k8s.gcr.io/pause                   3.1     da86e6ba6ca1  2 years ago        749 kB

```

Reply:
I would prefer not to go with some arbitrary name/description, as we really want to indicate that there is no name.

Reply:
The CLI is not i18n'ed, though. Well, it's not a very exposed feature, so if you want to keep `<none>` for now, that's ok with me.